packages/orbit-components/src/LinkList/index.tsx

```diff
@@ -99,24 +15,34 @@ const LinkList = ({
   children,
   dataTest,
 }: Props) => (
-  <StyledLinkList
-    $indent={indent}
-    $direction={direction}
+  <ul
     data-test={dataTest}
-    $legacy={legacy}
-    $spacing={spacing}
+    className={cx(
+      "flex",
+      "w-full",
+      "m-0",
+      "list-none",
+      "text-normal",
+      indent ? "ps-xxs" : "p-0",
```

Comment:
As said [here](https://github.com/kiwicom/orbit/pull/4083#discussion_r1360687872), please revert this